bin/bin.js

```diff
@@ -3,11 +3,18 @@ var lzString = require('../libs/lz-string.js');
 var fs = require('fs');
 
 if (process.argv.length < 3) {
-  console.error('Usage: lz-string <input_file>');
+  console.error('Usage: lz-string [funcName] <input_file>');
   process.exit(1);
 }
 
-console.log(lzString.compress(fs.readFileSync(process.argv[2], {
+var funcName = 'compress',
+    filePath = process.argv[2];
+
+if (process.argv.length == 4) {
+    funcName = process.argv[2];
```

Comment:
This would/could effectively expose any method available. If we go with having `compress` and `decompress` as the "command" (in `process.argv[2]`), then that may make this a little more consistent overall.